From 49ecc5380557e4bb7759350da0ef88977eb073c4 Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Thu, 21 Jan 2016 13:31:25 +0800
Subject: [PATCH 593/602] MA-7541 Use "eth0" in sabresd_7d platform

In imx7d-sdb-revB the eth1 is also enabled. So
that the ConnectivityService may enumerate eth1
first and make it as Ethernet port. But we have
no configuration for eth1.
So fix eth0 to be the default Ethernet port
in sabresd_7d platform.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml b/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
index 34f7d63..07e579d 100644
--- a/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/sabresd_7d/overlay/frameworks/base/core/res/res/values/config.xml
@@ -28,6 +28,9 @@
        <item>rndis0</item>
      </string-array>
 
+     <!--For Android we support eth0 now -->
+     <string translatable="false" name="config_ethernet_iface_regex">eth0</string>
+
     <!-- List of regexpressions describing the interface (if any) that represent tetherable
          Wifi interfaces.  If the device doesn't want to support tethering over Wifi this
          should be empty.  An example would be "softap.*" -->
-- 
1.8.0

